test/run_nighthawk_bazel_coverage.sh

```diff
@@ -57,7 +57,7 @@ then
   COVERAGE_VALUE=$(llvm-cov export "${COVERAGE_BINARY}" -instr-profile="${COVERAGE_DATA}" \
     -ignore-filename-regex="${COVERAGE_IGNORE_REGEX}" -summary-only | \
     python3 -c "import sys, json; print(json.load(sys.stdin)['data'][0]['totals']['lines']['percent'])")
-  COVERAGE_THRESHOLD=98.0
+  COVERAGE_THRESHOLD=97.7
```

Comment:
This is a pretty high coverage threshold, and is likely to result in more edits in the future. Is the idea that whenever the threshold is reduced, we want to make the developer actively note it by lowering this value? Or could this value be reduced to a more reasonable longterm threshold?

Reply:
So far the strategy was to keep this borderline high indeed, to queue authors/reviewers when something is up with coverage.  Personally I like that: there's some upkeep to this, but figuring out what changed and what needs to be done with respect to coverage is really easy. And we get auto-queued when something is off. So I bump in decrease coverage as-needed. E.g. the in flight grpc stuff will give a boost to coverage (that PR has `98.9%`), I had to lower it while splitting that up into pieces to not make master fail.

But as far as I am concerned, this isn't written in stone. I can imagine that with more people working on this might get uncomfortable. I believe @htuch and me have been aiming for `97.5%` initially.
It's worth noting that the code base is relatively small, so small changes in covered lines and even code-base size make a relatively big difference.